src/Tvheadend.cpp

```diff
@@ -2470,6 +2470,24 @@ void CTvheadend::ParseRecordingAddOrUpdate ( htsmsg_t *msg, bool bAdd )
   if ((str = htsmsg_get_str(msg, "fanartImage")) != NULL)
     rec.SetFanartImage(GetImageURL(str));
 
+  /* Extra checks for HTSP v32+ */
```

Comment:
Please remove this comment. It is more then obvious what the following code line does.